spec/rotate_matrix_spec.rb

```
@@ -54,6 +54,40 @@ def suspicious_rotate_matrix_counter_clockwise(matrix)
 #
 #   return result
 # end
+=======
```

RuboCop (1.76.1)

[fatal] unexpected token tEQQ
(Using Ruby 3.2 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)